internal/eigenState/avsOperators/avsOperators.go

```diff
@@ -106,11 +102,11 @@ func (a *AvsOperatorsModel) GetModelName() string {
 //
 // Returns the map and a reverse sorted list of block numbers that can be traversed when
 // processing a log to determine which state change to apply.
-func (a *AvsOperatorsModel) GetStateTransitions() (types.StateTransitions[AccumulatedStateChange], []uint64) {
-	stateChanges := make(types.StateTransitions[AccumulatedStateChange])
+func (a *AvsOperatorsModel) GetStateTransitions() types.StateTransitions {
+	stateTransitions := make(types.StateTransitions)
 
 	// TODO(seanmcgary): make this not a closure so this function doesnt get big an messy...
-	stateChanges[0] = func(log *storage.TransactionLog) (*AccumulatedStateChange, error) {
+	stateTransitions[0] = func(log *storage.TransactionLog) (interface{}, error) {
```

Comment:
Why remove the typed return?

Reply:
This is done to make it play nice with BaseEigenState. If we end up using a pure helper method, then we can make it generic again, however the typed return value is also beacons untyped in HandleLog anyway